systemtest/src/main/java/io/strimzi/systemtest/TestTags.java

```diff
@@ -0,0 +1,224 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.systemtest;
+
+/**
+ * Defines constants used as tags in the tests. These tags categorize and filter
+ * tests based on their purpose, stability, and environment, enabling selective execution of tests
+ * for scenarios like regression, scalability, upgrades, and performance.
+ */
+public interface TestTags {
+
+    /**
+     * Feature gate related constants
+     */
+    // No Feature gates kept for STs at this moment
+
+    /**
+     * Default value which allows execution of tests with any tags
+     */
+    String DEFAULT_TAG = "all";
+
+    /**
+     * Tag for acceptance tests, which can be triggered manually for each push/pr/merge on Azure
+     */
+    String ACCEPTANCE = "acceptance";
+
+    /**
+     * Tag for regression tests which are stable.
+     */
+    String REGRESSION = "regression";
+
+    /**
+     * Tag for upgrade tests.
+     */
+    String UPGRADE = "upgrade";
+
+    /**
+     * Tag for KRaft to KRaft tests.
+     */
+    String KRAFT_UPGRADE = "kraftupgrade";
+
+    /**
+     * Tag for olm upgrade tests
+     */
+    String OLM_UPGRADE = "olmupgrade";
+
+    /**
+     * Tag for smoke tests
+     */
+    String SMOKE = "smoke";
+
+    /**
+     * Tag for Kafka smoke tests
+     */
+    String KAFKA_SMOKE = "kafkasmoke";
+
+    /**
+     * Tag for sanity tests
+     */
+    String SANITY = "sanity";
+
+    /**
+     * Tag for tests, which results are not 100% reliable on all testing environments.
+     */
+    String FLAKY = "flaky";
+
+    /**
+     * Tag for scalability tests
+     */
+    String SCALABILITY = "scalability";
+
+    /**
+     * Tag for tests containing scaling of particular component (scaling up and down)
+     */
+    String COMPONENT_SCALING = "componentscaling";
+
+    /**
+     * Tag for tests, which are working only on specific environment and we usually don't want to execute them on all environments.
+     */
+    String SPECIFIC = "specific";
```

Comment:
I guess we can remove this one, it's just one test suite which uses it and it contains just one test - the name of the suite is `SpecificST`, which means that you can specify just this suite instead of the group